lib/internal/test_runner/mock.js

```diff
@@ -157,16 +161,30 @@ class MockTracker {
         'options.setter', setter, "cannot be used with 'options.getter'"
       );
     }
+    function getOriginalObject(descriptor) {
+      let original;
+
+      if (getter) {
+        original = descriptor?.get;
+      } else if (setter) {
+        original = descriptor?.set;
+      } else {
+        original = descriptor?.value;
+      }
+
+      return original;
+    }
+    let descriptor = ObjectGetOwnPropertyDescriptor(object, methodName);
+    let original = getOriginalObject(descriptor);
 
-    const descriptor = ObjectGetOwnPropertyDescriptor(object, methodName);
-    let original;
+    // classes instances
+    if (typeof original !== 'function') {
+      descriptor = ObjectGetOwnPropertyDescriptor(
+        ObjectGetPrototypeOf(object),
+        methodName
+      );
```

Comment:
This would probably need to be recursive, to account for subclasses of subclasses.

```js
class A { static method(){} }
class B extends A {}
class C extends B {}
```

Reply:
loved it! Thank you! Just updated the code with it